api/authorizer/client.go

```diff
@@ -143,3 +143,107 @@ func (auth *Client) SignPrincipalKey(groupID, keyID string, credential *Credenti
 
 	return signature, err
 }
+
+// ExtenderCACertificates gets authorizer's extender CA certificates
+func (auth *Client) ExtenderCACertificates(accessGroupID string) ([]CA, error) {
+	certificates := []CA{}
+	filters := Params{
+		AccessGroupID: accessGroupID,
+	}
+
+	_, err := auth.api.
+		URL("/authorizer/api/v1/extender/cas").
+		Query(&filters).
+		Get(&certificates)
+
+	return certificates, err
+}
+
+// ExtenderCACertificate gets authorizer's extender CA certificate
+func (auth *Client) ExtenderCACertificate(id string) (*CA, error) {
+	certificate := &CA{}
+
+	_, err := auth.api.
+		URL("/authorizer/api/v1/extender/cas/%s", url.PathEscape(id)).
+		Get(&certificate)
+
+	return certificate, err
+}
+
+// ExtenderCertificateRevocationList gets authorizer CA's certificate revocation list
+func (auth *Client) ExtenderCertificateRevocationList(filename, id string) error {
+	err := auth.api.
+		URL("/authorizer/api/v1/extender/cas/%s/crl", url.PathEscape(id)).
+		Download(filename)
+
+	return err
+}
+
```

Comment:
Naming convention suggests a resource should be returned. Here nothing is returned (except err) so the function name could be DownloadExtenderCertificateCRL() to indicate an action (downloading).